app/src/main/res/values/strings.xml

```diff
@@ -153,6 +153,7 @@
     <string name="no">No</string>
     <string name="found">found</string>x
 
+    <string name="password_key">Password</string>
```

Comment:
As this string will act as a key then it should not be translated to other languages, so please add` translatable="false"`

Reply:
Sorry for not paying attention to that, changes done.